api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java

```diff
@@ -21,7 +21,8 @@
 
 /**
  * Injects and extracts a value as text into carriers that travel in-band across process boundaries.
- * Values are often encoded as RPC/HTTP request headers.
+ * Encoding is expected to conform to the HTTP/1.1 Header Field semantics. Values are often encoded
```

Comment:
is there difference with 1.0 and 2.0? Perhaps this comment narrowing the use of this class to much

Reply:
2.0 adds binary protocol, and that's pretty much it, IIRC

Reply:
So should we specify that this works only for HTTP/1.1? Can comment be more generic and say simply HTTP?

Reply:
I see your point. Yeah, totally, lets go the generic way.